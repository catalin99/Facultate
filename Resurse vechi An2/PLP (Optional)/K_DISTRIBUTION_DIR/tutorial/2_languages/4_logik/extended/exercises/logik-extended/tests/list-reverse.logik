reverse([],L,L).
reverse([H|T],L1,L2) :- reverse(T,[H|L1],L2).

reverse(L,R) :- reverse(L,[],R).

?- reverse([1,2,3],L).
