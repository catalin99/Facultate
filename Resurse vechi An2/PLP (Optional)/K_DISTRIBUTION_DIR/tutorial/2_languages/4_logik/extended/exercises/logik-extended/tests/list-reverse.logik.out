Search results:

Solution 1:
Solution:Map -->
"L" |-> [ 3 | ([ 2 | ([ 1 | ([ .List{","} ]) ]) ]) ]
